GitUI/UserControls/RevisionGrid.cs

```diff
@@ -863,7 +863,7 @@ public List<GitRevision> GetSelectedRevisions()
             return GetSelectedRevisions(null);
         }
 
-        public string DescribeRevision(GitRevision revision)
+        public string DescribeRevision(GitRevision revision, int maxLength = 50)
```

Comment:
this will break this
https://github.com/gitextensions/gitextensions/blob/c133bf68944be5e68fcdc66dc7dcdc4c792c222e/GitUI/CommandsDialogs/RevisionDiff.cs#L195